libraries/AP_HAL_ChibiOS/Scheduler.cpp

```diff
@@ -58,6 +58,14 @@
 extern AP_IOMCU iomcu;
 #endif
 
+#ifdef IOMCU_FW
```

Comment:
This is only with H7 I think? Probably should add that to the define.

Reply:
it gets removed for other boards that don't use it by compiler. As shown by no size difference.